jobsets/tests/make-test.nix

```diff
@@ -0,0 +1,5 @@
+f: { system ? builtins.currentSystem, ... } @args:
```

Comment:
We could get rid of this file by calling `import <nixpkgs/nixos/tests/make-test.nix> ...` for each test